helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -957,18 +957,27 @@ private RollingUpdateOp rollingUpdateAwaitRunning(final ZooKeeperClient client,
 
   private boolean isRolloutTimedOut(final ZooKeeperClient client,
                                     final DeploymentGroup deploymentGroup) {
+    final String groupName = deploymentGroup.getName();
+    final long groupTimeoutSetting = deploymentGroup.getRolloutOptions().getTimeout();
+    final long secondsSinceDeploy;
     try {
-      final String statusPath = Paths.statusDeploymentGroupTasks(deploymentGroup.getName());
-      final long secondsSinceDeploy = MILLISECONDS.toSeconds(
+      final String statusPath = Paths.statusDeploymentGroupTasks(groupName);
+      secondsSinceDeploy = MILLISECONDS.toSeconds(
           System.currentTimeMillis() - client.getNode(statusPath).getStat().getMtime());
-      return secondsSinceDeploy > deploymentGroup.getRolloutOptions().getTimeout();
     } catch (KeeperException e) {
       // statusPath doesn't exist or some other ZK issue. probably this deployment group
       // was removed.
-      log.warn("error determining deployment group modification time: {} - {}",
-          deploymentGroup.getName(), e);
+      log.warn("error determining deployment group modification time: {}", groupName, e);
       return false;
     }
+
+    if (secondsSinceDeploy > groupTimeoutSetting) {
+      log.info("rolling-update on deployment-group name={} has timed out after "
```

Comment:
`info` to match the log message that is logged when a rollout fails due to timeout here https://github.com/spotify/helios/blob/f02446657f06c4fa8f7aad9a23bf87d429fb80db/helios-services/src/main/java/com/spotify/helios/rollingupdate/RollingUpdateOpFactory.java#L198-L199